tests/copilot_helper/utils/fixtures/addons_files/opensearch_addons_bad_data.yml

```diff
@@ -0,0 +1,62 @@
+my-opensearch-1:
+  type: opensearch
+  nonsense: abc
+
+my-opensearch-2:
+  type: opensearch
+  deletion-policy: 27
+
+my-opensearch-3:
+  type: opensearch
+  environments: False
+
+my-opensearch-4:
+  type: opensearch
+  environments:
+    dev:
+      opensearch-plan: large
+
+my-opensearch-5:
+  type: opensearch
+  environments:
+    dev:
+      plan: largish
+
+my-opensearch-6:
+  type: opensearch
+  environments:
+    dev:
+      replicas: 6
+
+my-opensearch-7:
+  type: opensearch
+  environments:
+    dev:
+      instance: medium
+
+my-opensearch-8:
+  type: opensearch
+  environments:
+    dev:
+      engine: 7.3
+
+my-opensearch-9:
+  type: opensearch
+  environments:
+    dev:
+      volume_size: 9
+
+my-opensearch-10:
+  type: opensearch
+  environments:
+    dev:
+      volume_size: 512
+
+my-opensearch-11:
+  type: opensearch
+  environments:
+    dev:
+      deletion-policy: Snapeshot
```

Comment:
```suggestion
      deletion-policy: Snapshot
```

Reply:
This is deliberate bad data for the test